src/propTypes.js

```diff
@@ -260,10 +260,7 @@ const DATA_SHAPE = shape({
   colors: objectOf(
     oneOfType([
       func,
-      shape({
-        pattern: arrayOf(string),
-        tiles: func,
-      }),
+      PropTypes.string,
```

Comment:
This is a good change, but I fear the same issue exists above with the singular `color`. Would you mind making this a shared type and reusing it?

```js
const DATA_COLOR_SHAPE = oneOfType([func, string]);
...
color: DATA_COLOR_SHAPE,
colors: objectOf(DATA_COLOR_SHAPE),
```

Reply:
Yup, seems like color shape was outdated, but according to [the api](https://naver.github.io/billboard.js/release/latest/doc/Options.html#.data%25E2%2580%25A4color) it should be a function, not the same shape as colors. BTW i was not using the destructured proptypes.

I amended the commit chaging that prop as well, unfortunately the object cannot be reused.

@planttheidea Ty for taking your time to review